src/components/blocks/software/Pre.tsx

```diff
@@ -163,7 +168,14 @@ const Pre = ({
 
 export default Pre;
 
-const cleanIfLanguageHasSDKInterface = (language: string) =>
+export const cleanIfLanguageHasSDKInterface = (language: string) =>
   language.includes(`${REALTIME_SDK_INTERFACE}_`) || language.includes(`${REST_SDK_INTERFACE}_`)
     ? language.split('_', 2)[1]
     : language;
+
+const getLanguagesSDKInterface = (allLanguage: string[], selectedSDKInterface: string) =>
+  allLanguage
+    .map((language) => (language.includes(`${selectedSDKInterface}_`) ? language : ''))
+    .filter(function (n: string) {
```

Comment:
nitpick: could we use `s` instead of `n` here?

Reply:
@artismarti  just updated and simplified it

Reply:
still seeing `s` here, @aralovelace
Do you need to push?

Reply:
@artismarti  you mean I replace n to s?